packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount.jsx

```diff
@@ -95,15 +95,24 @@ const Amount = ({
 
     const getBasisList = () => basis_list.map(item => ({ text: item.text, value: item.value }));
 
+    const setToolTipContent = () => {
```

Comment:
Why do we have different function/component with the same name?
also need to change name to `setTooltipContent`

Reply:
There are 2 popovers - 
On `amount.jsx`, the Fieldset component will handle rendering of the popup bubble.
In `trade-params-mobile.jsx`, the Popover component is used to render the same effect.